### PR TITLE
Design of README markdown file.

### PR DESCRIPTION
**Pull request by BrunildaDosti**
Mantis issue:
https://mantis.phplist.org/view.php?id=19492

*Images are uploaded to phpListSite repo.

### CODE REVIEW

**Comment by BrunildaDosti** [comment]
> The whole code is written in HTML now, not markdown. Was this intended?

The code was written using https://jbt.github.io/markdown-editor
The code is HTML and markdown syntax too.

File: README-new-version.md [hunk 1]
```diff
@@ -3,19 +3,16 @@
...
+[![Build Status](https://travis-ci.org/phpList/phplist3.svg?branch=master)](https://travis-ci.org/phpList/phplist3)
+[![Stable release](https://img.shields.io/badge/stable-3.4.1-blue.svg)](https://sourceforge.net/projects/phplist/files/phplist/)
+[![License](https://poser.pugx.org/phplist/phplist4-core/license.svg)](https://www.gnu.org/licenses/agpl-3.0.en.html)
+
 <p>Fully functional Open Source email marketing manager for creating, sending, integrating, and analysing email campaigns and newsletters:<a href="https://www.phplist.org">https://www.phplist.org</a></p>
 <p>phpList includes analytics, segmentation, content personalisation, bounce processing, plugin-based architecture, and multiple APIs. Used in 95 countries, available in 20 languages, and used to send more than 25 billion campaign messages in 2018.</p>
 <div align="center" width="100%">
-  <a href="https://www.phplist.org/">Community |</a> 
-  <a href="https://www.phplist.com/hosted?utm_source=hostedvsdownload&utm_medium=link&utm_campaign=phpList">Hosted |</a> 
+  <a href="https://discuss.phplist.org/">Community |</a>
```
Comment (samtuke) [inline README-new-version.md]: These links were correct before, please revert changes to these two lines

**Comment by samtuke** [comment]
Is this now redundant due to #532?

**Comment by samtuke** [comment]
> Is this now redundant due to #532?

Turns out yes, it is. Closing.

**Pull request closed without merging**